client/db.go

```diff
@@ -265,9 +265,9 @@ type RequestResult struct {
 	// GQL contains the immediate results of the GQL request.
 	GQL GQLResult
 
-	// Pub contains a pointer to an event stream which channels any subscription results
-	// if the request was a GQL subscription.
-	Pub *events.Publisher[events.Update]
```

Comment:
discussion: I think you are removing functionality here - IIRC the events package stuff allows for multiple concurrent readers of the stream.  The simple Go chan that you are replacing it lacks this functionality and will mean any concurrent readers will be competing with each other for each item.

The CLI/http clients are unaffected, but the embedded Go client is.

Reply:
It makes more sense to me that we assume one reader per request by default. The consumer of the channel then has the option to decide if they want to create an `events.Publisher` and publish the channel results to it.